docs/hardhat.md

```diff
@@ -37,6 +37,8 @@ If a hardhat project is shared through remixd and `localhost` workspace is loade
 
 ![](images/a-hardhat-compilation.png)
 
+There is an info icon along side the label which redirects to a specific section of Remix official documentation that explains how to use Hardhat compilation.
+
 One can check this box to run the compilation for Hardhat along with the Remix using the compiler configuration in `Solidity Compiler` plugin.
```

Comment:
With you update - its not so clear what the "this box" is referring to - maybe update it to: 
Check the **Enable Hardhat Compilation** box to ...

Reply:
updated